addons/mail_plugin/controllers/mail_plugin.py

```diff
@@ -53,6 +53,71 @@ def res_partner_enrich_and_create_company(self, partner_id):
 
         return response
 
+    @http.route('/mail_plugin/partner/enrich_and_update_company', type='json', auth='outlook', cors='*')
+    def res_partner_enrich_and_update_company(self, partner_id):
+        """
+        Enriches an existing company using IAP
+        """
+        partner = request.env['res.partner'].browse(partner_id)
+        if not partner.is_company:
+            return {'error': _('Contact must be a company')}
```

Comment:
Just asking, why do we sometimes have a key and sometimes a translated string ?

Reply:
The old plugin version has that, don't know why.
I think that this is inconsistent and that error messages unless displayed in the front end should not be translated, I was intending to force this behavior for all messages in the cleanup task.

Reply:
Ok, just add a note about that in your cleaning pad indeed, so that we may re-do a whole sanity check in mail plugin afterwards :) .